actionRouter.js

```diff
@@ -0,0 +1,81 @@
+const express = require("express");
+
+const Action = require("./data/helpers/actionModel");
+
+const actionRouter = express.Router();
+
+// Get all
+
+//Get actions by action_id
+
+//Post new action to specific action by action_id
+
+//Put action by id
+
+//Delete action by ID
+
+// Get all
+
+actionRouter.get("/", async (req, res) => {
+  try {
+    const action = await Action.get();
+    res.status(200).json({ action });
+  } catch (error) {
+    res.status(500).json({ message: "Error retrieving actions" });
+  }
+});
+
+//Get actions by action_id
+
+actionRouter.get("/:id", async (req, res) => {
+  try {
+    const action = await Action.get(req.params.id);
+    res.status(200).json({ action });
```

Comment:
you could add a conditional in here checking if !action and do a res.status(404).json({message: "action not found"})